samples/subsys/ipc/ipc_service/static_vrings/src/main.c

```diff
@@ -179,16 +179,23 @@ K_THREAD_DEFINE(ipc0B_thread_id, STACKSIZE, ipc0B_entry, NULL, NULL, NULL, PRIOR
 
 /*
  * ==> THREAD 1 (IPC instance 1) <==
+ *
+ * NOTE: This instance is using the NOCOPY copability of the backend.
  */
 
+static struct ipc_ept ipc1_ept;
+static void *recv_data;
+
 static void ipc1_ept_bound(void *priv)
 {
 	k_sem_give(&ipc1_bound_sem);
 }
 
 static void ipc1_ept_recv(const void *data, size_t len, void *priv)
 {
-	ipc1_received_data = *((uint8_t *) data);
+	ipc_service_hold_rx_buffer(&ipc1_ept, (void *) data);
```

Comment:
handle error code. In future there might be backend that does not support it.

Reply:
ack